package-lock.json

```diff
@@ -1852,19 +1852,6 @@
         "@babel/core": "^7.0.0-0"
       }
     },
-    "node_modules/@babel/plugin-transform-object-assign": {
-      "version": "7.18.6",
-      "license": "MIT",
-      "dependencies": {
-        "@babel/helper-plugin-utils": "^7.18.6"
-      },
-      "engines": {
-        "node": ">=6.9.0"
-      },
-      "peerDependencies": {
-        "@babel/core": "^7.0.0-0"
-      }
-    },
```

Comment:
Why do we have these changes? I don't see anything changing in `package.json`

Reply:
I think a PR that just adds some feature and does not modify any packages should not introduce any changes to `package-lock` - even if these changes are on main. That's messy.

I suggest you drop/revert any changes to lockfile @jnowakow

Reply:
I reverted that change as @Kicu advised